tests/vmi_multus_test.go

```diff
@@ -874,6 +866,45 @@ var _ = Describe("SRIOV", func() {
 		It("[test_id:3957]should connect to another machine with sriov interface over IPv6", func() {
 			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
+
+		Context("VLANed SR-IOV network", func() {
+			var vlanedVMI1 *v1.VirtualMachineInstance
+			cidrVlaned1 := "192.168.0.1/24"
+			cidrVlaned2 := "192.168.0.2/24"
+			cidrFlat := "192.168.0.3/24"
+			BeforeEach(func() {
+				createNetwork("sriov-vlan", tests.NamespaceTestDefault, sriovConfVlanCRD)
+				createNetwork("sriov-no-vlan", tests.NamespaceTestDefault, sriovConfNoVlanCrd)
+				vlanedVMI1 = createSriovVMI([]string{"sriov-vlan"})
+
+				// manually configure IP/link on sriov interfaces because there is
+				// no DHCP server to serve the address to the guest
+				configInterface(vlanedVMI1, "eth1", cidrVlaned1, "#")
+			})
+
+			AfterEach(func() {
+				deleteNetwork("sriov-vlan", tests.NamespaceTestDefault)
+				deleteNetwork("sriov-no-vlan", tests.NamespaceTestDefault)
+			})
+
+			It("[test_id:3958]should be able to ping between two VMs with the same VLAN over SRIOV network", func() {
+				vlanedVMI2 := createSriovVMI([]string{"sriov-vlan"})
+				configInterface(vlanedVMI2, "eth1", cidrVlaned2, "#")
+
+				By("pinging between vlanedVMI1 and vlanedVMI2 over vlan")
+				err = testPingFromVMI(vlanedVMI1, cidrToIP(cidrVlaned2), "#")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s should be reachable", vlanedVMI2.Name)
+			})
+
+			It("[test_id:3958]should NOT be able to ping between Vlaned VM and a flat VM", func() {
```

Comment:
Is "flat vm" is a know terminology? I would call it "non-Vlaned VM"

Reply:
changed

Reply:
The `It` still contains "flat VM"

Reply:
oops  missed that. Tnx